recipes/alembic/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+from conans import ConanFile, CMake, tools
+import os
+import shutil
+
+
+class AlembicConan(ConanFile):
+    name = "alembic"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/alembic/alembic"
+    description = "Open framework for storing and sharing scene data."
+    topics = ("3d", "scene", "geometry", "graphics")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_hdf5": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_hdf5": False
+    }
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["CMakeLists.txt", "AlembicIlmBase.cmake"]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("openexr/2.5.7")
+        if self.options.with_hdf5:
+            self.requires("hdf5/1.12.0")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["USE_ARNOLD"] = False
+        self._cmake.definitions["USE_MAYA"] = False
+        self._cmake.definitions["USE_PRMAN"] = False
+        self._cmake.definitions["USE_PYALEMBIC"] = False
+        self._cmake.definitions["USE_BINARIES"] = False
+        self._cmake.definitions["USE_EXAMPLES"] = False
+        self._cmake.definitions["USE_HDF5"] = self.options.with_hdf5
+        self._cmake.definitions["USE_TESTS"] = False
+        self._cmake.definitions["ALEMBIC_BUILD_LIBS"] = True
+        self._cmake.definitions["ALEMBIC_ILMBASE_LINK_STATIC"] = True # for -DOPENEXR_DLL, handled by OpenEXR package
+        self._cmake.definitions["ALEMBIC_SHARED_LIBS"] = self.options.shared
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _patch_sources(self):
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "./cmake/AlembicIlmBase.cmake",
+            "../AlembicIlmBase.cmake")
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "CMAKE_CXX_STANDARD 14",
+            "CMAKE_CXX_STANDARD 11")
```

Comment:
Can you explain the motivation behind this? it's unusual

Reply:
Alembic library is written in C++11 as noted in the documentation. However, if C++ standard is not explicitly provided through `-DCMAKE_CXX_STANDARD=` then the library falls back to C++14 requirement. This is due to [VFX Reference Platform](https://vfxplatform.com/) which gives specific GCC version, glibc version, C++ standard, dependency library versions etc. to use depending on reference platform year chosen.

In Conan the library is supposed to be compiled with many different compilers and versions so I think the VFX platform spec cannot be applied. As the library is C++11 compliant it did not seem necessary to limit building to only C++14-capable compilers.